data/tags.json

```
@@ -173,6 +173,15 @@
       "<https://archive.org/details/termux-repositories-legacy>"
     ]
   },
+  { "name": "linux",
+    "aliases": [],
```

I think `!tag distro` would be a good alias to add for this tag.
```suggestion
    "aliases": ["distro"],
```